modules/test/playwright/tests/notification-web/main/emailNotificationTemplate.spec.ts

```diff
@@ -126,6 +138,73 @@ test.describe('Email notification template', () => {
 		);
 	});
 
+	test('can be edited correctly', async ({
+		emailNotificationTemplatePage,
+		notificationTemplatesPage,
+	}) => {
+		await notificationTemplatesPage.goto();
+
+		await notificationTemplatesPage
+			.getFrontEndDatasetItemLocator(notificationTemplate.name)
+			.click();
+
+		const editedNotificationTemplateInfo = {
+			bcc: getRandomString(),
+			cc: getRandomString(),
+			description: getRandomString(),
+			recipients: getRandomString(),
+			senderAddress: getRandomString(),
+			senderName: getRandomString(),
+			subject: getRandomString(),
+			term: getRandomString(),
+		};
+
+		const editedNotificationTemplateName = getRandomString();
+
+		await emailNotificationTemplatePage.fillNotificationTemplateInfo(
+			editedNotificationTemplateName,
+			editedNotificationTemplateInfo
+		);
+
+		await emailNotificationTemplatePage.saveButton.click();
+
+		await notificationTemplatesPage
+			.getFrontEndDatasetItemLocator(editedNotificationTemplateName)
+			.click();
+
+		await expect(emailNotificationTemplatePage.basicInfoName).toHaveValue(
```

Comment:
For all these expects, use `expect.soft` instead. After all of them, at the end of the test, do `expect(test.info().errors).toHaveLength(0);`. This way, if any of the assertion fails the test will keep running. This will allow us to have a full picture of what is not working. `test.info().errors` will show all failed assertions at the end. Try for yourself when modifying.